README.md

```diff
@@ -12,20 +12,25 @@ You can find the deployed project at [https://a.humanrightsfirstdocdb.dev/](http
 
 
 
-|[Alex Lucchesi](https://github.com/lucchesia7) 
-[Alyssa Murray](https://github.com/dagtag)  
-[Greg Engelmann](https://github.com/engegreg) 
-[Hunter Jordan](https://github.com/Hunter-Jordan)
-[Peter Rockwood](https://github.com/prockwood)
-[Joshua Aurajo](https://github.com/joshua-aurajo)
+[Alex Lucchesi](https://github.com/lucchesia7)   
+
+[Alyssa Murray](https://github.com/dagtag)    
+
+[Greg Engelmann](https://github.com/engegreg)   
+
+[Hunter Jordan](https://github.com/Hunter-Jordan)  
+
+[Peter Rockwood](https://github.com/prockwood)  
+
+[Joshua Aurajo](https://github.com/joshua-aurajo)  
 
 
 <br>
 <br>
 
-![MIT](https://img.shields.io/packagist/l/doctrine/orm.svg)
-![Python](https://docs.python.org/3/)
```

Comment:
You are right in python doesn't require a license, but I still think keeping the python box there is beneficial as we can specify which python version at an eyes glance.

Reply:
That is fair, I re added it at the bottom of the license/style section.